proposals/v3/monitoring.md

```diff
@@ -0,0 +1,66 @@
+# Monitoring & Metrics
+## Summary
+Adding support for metrics export on Synse services grants additional observability
+into how the system is operating. Having metrics for service performance, bandwidth,
+latency, etc. will allow us to more easily identify trouble points and could lend to
+mitigation of issues before they become issues. 
+
+## High Level Work Items
+- Add configuration to Synse Server for metrics export
+- Define and collect a set of metrics for Synse Server to export
+- Add configuration to Synse SDK for metrics export
+- Define and collect a set of metrics for Synse SDK to export
+- Determine the means by which plugins will export their metrics
+
+## Proposal
+For both Synse Server and Synse Plugins (aka. the Plugin SDK), we will support the
+collection and export of metrics for Prometheus.
+
+This proposal is only to support metrics export by services; it makes no attempt
+to describe how metrics are collected, stored, and analyzed from these services.
+
+### Synse Server
+Synse Server will export its collected metrics via its HTTP API at the versionless
+`/metrics` endpoint.
```

Comment:
Should `/metrics` endpoint be documented in the API doc too?

Reply:
It probably should be. The details of this are still a little unknown to me, but I'll add a note that this exists in the API doc.